etc/notebook/kernels/root/kernel.json.in

```diff
@@ -2,7 +2,7 @@
  "language": "c++",
  "display_name": "ROOT C++",
  "argv": [
-  "python",
+  "python@PYTHON_VERSION_MAJOR@.@PYTHON_VERSION_MINOR@",
```

Comment:
Shouldn't this be `${PYTHON_EXECUTABLE}`? For example, on macOS you may have Homebrew Python vs the system Python with the same version, and just `python2.7` or `python3.5` may still pick the wrong one (i.e. `/usr/bin/python2.7` instead of `/usr/local/bin/python2.7` from Homebrew).

Reply:
After some discussion with @amadio , we agreed to not include the full path. Like this, a Python installation other than the one in the `kernel.json` file can be used to run the kernel without rewriting manually the `kernel.json`. Of course, such other installation needs to be compatible with the one that ROOT was built against (also compiled with the same compiler).